index.bs

```diff
@@ -1272,11 +1272,83 @@ spec: secure-contexts; urlPrefix: https://www.w3.org/TR/secure-contexts/
           &lt;/script>
         </pre>
       </div>
+
+      <p>
+        The following example shows how to use {{decodingInfo()}} to query
+        WebRTC receive capabilities [[webrtc]].
+      </p>
+
+      <div class="example" highlight="javascript">
+        <pre>
+          &lt;script>
+            const contentType = 'video/VP8';
```

Comment:
nit: can you use a more elaborate contentType that includes profile and even other parameters for folks to know where those go? For example, the contentType equivalent of 

"mimeType": "video/H264",
"sdpFmtpLine": "level-asymmetry-allowed=1;packetization-mode=1;profile-level-id=42e01f"

Reply:
Done!